### PR TITLE
Spell check fixes

### PR DESCRIPTION
**Pull request by karthikeyann**
Regular spell check fixes in comments and docs.

### CODE REVIEW

**Review by galipremsagar — APPROVED** [review]

File: python/cudf/cudf/tests/test_datetime.py [hunk 1]
```diff
@@ -171,7 +171,7 @@ def test_dt_ops(data):
     assert_eq(pd_data > pd_data, gdf_data > gdf_data)
 
 
-# libgdf doesn't respect timezones
+# licudf doesn't respect timezones
```
Comment (charlesbluca) [inline python/cudf/cudf/tests/test_datetime.py:174]:
```suggestion
# libcudf doesn't respect timezones
```

**Review by charlesbluca — CHANGES_REQUESTED** [review]

**Review by charlesbluca — APPROVED** [review]

**Comment by karthikeyann** [comment]
rerun tests

**Comment by karthikeyann** [comment]
rerun tests

**Comment by karthikeyann** [comment]
rerun tests

**Comment by karthikeyann** [comment]
rerun tests

**Comment by karthikeyann** [comment]
@gpucibot merge

**Pull request merged**